app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Go through the Slack thread and point out to me what's wrong with the filenaming.

Reply:
Yes @yedhink.
I will change it to List.jsx